CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ if(NOT JQ_EXE)
   message(FATAL_ERROR "jq is required for output formatting")
 endif()
 
+find_program(PYTHON_EXE python3)
+if(NOT PYTHON_EXE)
```

Comment:
There is probably a way to existing cmake templating  logic for metadata but I'm fine with however we want to manage this.